draft-sharabayko-srt.md

```diff
@@ -1531,113 +1544,123 @@ The INDUCTION phase serves only to set a cookie on the Listener so that it
 - Handshake Type: INDUCTION
 - SRT Socket ID: SRT Socket ID of the Caller
 - SYN Cookie: 0.
+- There MUST be no HS extensions.
 
-The Destination Socket ID of the SRT packet header in this message is 0, which is
+The Destination SRT Socket ID of the SRT packet header in this message is 0, which is
 interpreted as a connection request.
 
 The handshake version number is set to 4 in this initial handshake.
 This is due to the initial design of SRT that was to be compliant with the UDT
 protocol {{GHG04b}} on which it is based.
 
+##### The Induction Response
+
 The Listener responds with the following:
 
-- Version: 5
-- Encryption Field: Advertised cipher family and block size
-- Extension Field: SRT magic code 0x4A17
+- HS Version: 5.
+- Encryption Field: Advertised cipher family and block size.
+- Extension Field: SRT magic code 0x4A17.
 - Handshake Type: INDUCTION
 - SRT Socket ID: Socket ID of the Listener
 - SYN Cookie: a cookie that is crafted based on host, port and current time
   with 1 minute accuracy to avoid SYN flooding attack {{RFC4987}}.
+- There MUST be no HS extensions.
 
 At this point the Listener still does not know if the Caller is SRT or UDT,
 and it responds with the same set of values regardless of whether the Caller is
 SRT or UDT.
 
-If the party is SRT, it does interpret the values in Version and Extension Field.
-If it receives the value 5 in Version, it understands that it comes from an SRT
-party, so it knows that it should prepare the proper handshake messages
-phase. It also checks the following:
+A legacy UDT party completely ignores the values reported in the HS Version and the Handshake Type field.  
+It is, however, interested in the SYN Cookie value, as this must be passed to the next 
+phase. It does interpret these fields, but only in the "conclusion" message.
 
-- whether the Extension Flags contains the magic value 0x4A17; otherwise the
-  connection is rejected. This is a contingency for the case where someone who,
-  in an attempt to extend UDT independently, increases the Version value to 5
-  and tries to test it against SRT;
+#### The Conclusion Phase
 
-- whether the Encryption Flags contain a non-zero
-  value, which is interpreted as an advertised cipher family and block size.
+##### The Conclusion Request
 
-A legacy UDT party completely ignores the values reported in Version and Handshake Type.
-It is, however, interested in the SYN Cookie value, as this must be passed to the next
-phase. It does interpret these fields, but only in the "conclusion" message.
+The SRT caller receives the Induction Response from the SRT listener. The SRT caller MUST check the Induction response from the SRT listener.
 
-#### The Conclusion Phase
+If the HS Version value is 5, the response came from SRT, and the handshake version 5 procedure is performed
+as covered further. 
+If the HS Version value is 4, the legacy handshake procedure can be applied if supported. The procedure is deprecated and is not covered here.
```

Comment:
`The procedure is deprecated and is not covered here.`

Is there a description in a previous version of the draft to which we can refer the reader?